targets/WindowsSdk/make.js

```diff
@@ -103,10 +103,10 @@ function getAuthParams(apiCall) {
     if (apiCall.url === "/Authentication/GetEntityToken")
         return "authKey, authValue";
     switch (apiCall.auth) {
-        case "None": return "U(\"\"), U(\"\")";
-        case "EntityToken": return "U(\"X-EntityToken\"), PlayFabSettings::entityToken";
-        case "SessionTicket": return "U(\"X-Authorization\"), PlayFabSettings::clientSessionTicket";
-        case "SecretKey": return "U(\"X-SecretKey\"), PlayFabSettings::developerSecretKey";
+        case "None": return "L\"\", L\"\"";
```

Comment:
Nonblocking - FWIW, I think mixing single/double quotes would make this a bit more readable. Not worth changing just the one line here by itself, but it is something to consider in the future.

```
case "None": return 'L"", L""';
```

Reply:
I would agree with you... except that Resharper complains if you do that.
And I'm a bit OCD about those stupid marks all over my screen distracting me.